### PR TITLE
feature/figured-bass-font

### PR DESCRIPTION
**Pull request by arthurian**
This PR adds the ability to parse chord labels and set the figured bass font for substrings wrapped in curly braces, sort of like a templating mechanism.

Example:

```
Input string: "dim{4e}/V"
At runtime output the chord label text like this:
   1. Render "dim" with default font.
   2. Render "4e" with special figured bass font.
   3. Render "/V" with default font.
```

@jazahn Can you review this? In particular, I'd like to pay special attention to the regular expression used to parse the chord label strings and the subsequent code to swap the fonts.

### CODE REVIEW

**Comment by arthurian** [comment]
@jazahn Can you review this?

File: harmony/static/js/src/components/music/stave_notater.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ define([
 		 * Defines the default font size.
 		 * @type {string}
 		 */
-		defaultFontSize: "14px",
+		defaultFontSize: "18px",
```
Comment (jazahn) [inline harmony/static/js/src/components/music/stave_notater.js]: is there any chance you can use em instead of px?

Comment (arthurian) [inline harmony/static/js/src/components/music/stave_notater.js]: Yeah, no particular reason using "px" here other than convenience.

**Comment by jazahn** [comment]
:+1:

**Pull request merged**